examples/objective-c/route_guide/ViewControllers.m

```diff
@@ -116,6 +123,9 @@ - (void)viewDidLoad {
 }
 
 - (void)viewDidAppear:(BOOL)animated {
+  self.outputLabel.text = @"RPC log:";
+  self.outputLabel.numberOfLines = 0;
+  self.outputLabel.font = [UIFont fontWithName:@"Helvetica Neue" size:8.0];
```

Comment:
I think we should be able to set these two properties via the UI builder, to get them out of the way too.